src/d3d11/meson.build

```diff
@@ -25,15 +25,25 @@ d3d11_src = [
   'd3d11_view_srv.cpp',
   'd3d11_view_uav.cpp',
 ]
-
+if not dxvk_wine_builtin
 d3d11_dll = shared_library('d3d11', d3d11_src,
   name_prefix         : '',
   link_with           : [ util_lib ],
-  dependencies        : [ dxvk_dep, dxgi_dep, dxbc_dep ],
+  dependencies        : [ declare_dependency(link_args: ['-ldxgi']), dxbc_dep, dxvk_dep ],
```

Comment:
Why is this change needed, i.e. just using `dxgi_dep` not enough?

Reply:
I don't tested dll-redirects with MinGW build, but if MinGW links libraries similar to winegcc than it should help (will check later). I have noticed no difference with generic MinGW build.